arbitrum-docs/devs-how-tos/how-to-estimate-gas.mdx

```diff
@@ -0,0 +1,142 @@
+---
+id: how-to-estimate-gas
+title: "How to estimate gas in Arbitrum"
+description: Learn how to estimate gas before submitting transactions.
+author: TucksonDev
```

Comment:
```suggestion
author: TucksonDev
content-type: how-to
```

no action required, just a future optimization - explicitly identifying a single content type can help us apply patterns and principles per type (like validating a schema)